docs/src/main/asciidoc/spring-di.adoc

```diff
@@ -334,6 +334,10 @@ The following table shows how Spring DI annotations can be converted to CDI and
 |Doesn't have a one-to-one mapping to a CDI annotation.
 |===
 
+== Spring DI API Configuration Reference
```

Comment:
Not sure why `API`?
```suggestion
== Spring DI Configuration Reference
```

Reply:
I went with the title of the page (Spring DI API) for consistency. But sure I can drop it.